### PR TITLE
[Text Generation][Tests] Text Generation Pipeline

### PR DESCRIPTION
**Pull request by dbogunowicz**
Adds basic testing to harden the most brittle parts of the pipeline.

### CODE REVIEW

File: tests/deepsparse/transformers/pipelines/test_text_generation.py
Comment (bfineran) [inline tests/deepsparse/transformers/pipelines/test_text_generation.py:59]: can't land until we update here

**Review by bfineran — CHANGES_REQUESTED** [review]
LGTM overall, 
1. let's move this away from the core pytest pathway as is right now since it will be both very long running and have a larger memory req
2. after landing, let's schedule a follow up to talk about how we can generalize and rearchitecture

**Comment by dbogunowicz** [comment]
Will merge this and introduces the appropriate changes on the feature branch level.

**Pull request merged**
